packages/next-templates/src/app/globals.css

```diff
@@ -218,6 +219,18 @@ utrecht-alert
   box-sizing: border-box;
 }
 
+.voorbeeld-toptask-layout {
+  display: flex;
+  justify-content: space-between;
+  gap: var(--voorbeeld-space-block-rabbit);
+  --utrecht-heading-3-margin-block-start: -5px;
+  --utrecht-heading-3-margin-block-end: var(--voorbeeld-space-block-rat);
+}
+
+.voorbeeld-paragraph-opening-times {
```

Comment:
Als `opening-times` een variant is, dan hoort het in BEM een modifier te zijn met `--`: `voorbeeld-paragraph--opening-times`